docs/migrations/v6-to-v7.md

```diff
@@ -173,3 +173,93 @@ Keeper function `CheckMisbehaviourAndUpdateState` has been removed since functio
+
+```diff
+message Header {
+  option (gogoproto.goproto_getters) = false;
+ 
+  - uint64              sequence        = 1;
+  - uint64              timestamp       = 2;
+  - bytes               signature       = 3;
+  - google.protobuf.Any new_public_key  = 4 [(gogoproto.moretags) = "yaml:\"new_public_key\""];
+  - string              new_diversifier = 5 [(gogoproto.moretags) = "yaml:\"new_diversifier\""];
+  + uint64              timestamp       = 1;
+  + bytes               signature       = 2;
+  + google.protobuf.Any new_public_key  = 3 [(gogoproto.moretags) = "yaml:\"new_public_key\""];
+  + string              new_diversifier = 4 [(gogoproto.moretags) = "yaml:\"new_diversifier\""];
+}
+```
+
+Similarly, the `Misbehaviour` protobuf message has been updated to remove the `client_id` field.
+
+```diff
+message Misbehaviour {
+  option (gogoproto.goproto_getters) = false;
+  
+  - string           client_id         = 1 [(gogoproto.moretags) = "yaml:\"client_id\""];
+  - uint64           sequence          = 2;
+  - SignatureAndData signature_one     = 3 [(gogoproto.moretags) = "yaml:\"signature_one\""];
+  - SignatureAndData signature_two     = 4 [(gogoproto.moretags) = "yaml:\"signature_two\""];
+  + uint64           sequence          = 1;
+  + SignatureAndData signature_one     = 2 [(gogoproto.moretags) = "yaml:\"signature_one\""];
+  + SignatureAndData signature_two     = 3 [(gogoproto.moretags) = "yaml:\"signature_two\""];
+}
+```
+
+### SignBytes
+
+Most notably, the `SignBytes` protobuf definition has been modified to replace the `data_type` field with a new field, `path`. The `path` field is defined as `bytes` and represents a serialized [ICS-24](https://github.com/cosmos/ibc/tree/main/spec/core/ics-024-host-requirements) standardized key path under which the `data` is stored.
+
+
+```diff
+message SignBytes {
+  option (gogoproto.goproto_getters) = false;
+
+  uint64 sequence    = 1;
+  uint64 timestamp   = 2;
+  string diversifier = 3;
+  - DataType data_type = 4 [(gogoproto.moretags) = "yaml:\"data_type\""];
+  + bytes path         = 4;
+  bytes data         = 5;
+}
+```
+
+The `DataType` enum and all associated data types have been removed, greatly reducing the number of message definitions and complexity in constructing the `SignBytes` message type. Likewise, solomachine implementations must now use the serialized `path` value when constructing `SignatureAndData` uses for signature verification of `SignBytes` data.
+
+```diff
+message SignatureAndData {
+  option (gogoproto.goproto_getters) = false;
+ 
+  bytes    signature                 = 1;
+  - DataType data_type                 = 2 [(gogoproto.moretags) = "yaml:\"data_type\""];
+  + bytes path                         = 2;
+  bytes    data                      = 3;
+  uint64   timestamp                 = 4;
```

Comment:
Just a nit, but maybe the `=` signs can be moved closer to the left...

Reply:
done, updated formatting!